### ISSUE TITLE
Enhance "choose account" dialog

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#5516]
![2020-02-24-165058](https://user-images.githubusercontent.com/5836855/75167638-f4c24400-5725-11ea-9d29-ef6d25e1bece.png)

- add avatar
- use display name @ server address instead of userId (same as in account drawer)

![image](https://user-images.githubusercontent.com/5836855/75167815-3f43c080-5726-11ea-9d58-fdcc701dc401.png)

@jancborchardt

**Comment by rohanprasadofficial** [comment]
@tobiasKaminsky  can i work/take this issue ?

**Comment by AndyScherzinger** [comment]
Hi @rohanprasadofficial, sure thing 👍
If you have any questions, please don't hesitate to ask @tobiasKaminsky, @ezaquarii or me. We are here to help 😊

**Comment by rohanprasadofficial** [comment]
@AndyScherzinger @tobiasKaminsky  i am working on it.

**Comment by szaimen** [comment]
Could that be based on #6116?

**Comment by AndyScherzinger** [comment]
Thanks for the ping @szaimen, the "right" PR is https://github.com/nextcloud/android/pull/6022#issuecomment-627123419 so fixed for 3.13.

**Comment by szaimen** [comment]
Yeah, hm not really...because for deep links it is not solved yet and still looks like the screenshot on the top.
(e.g. if you have more than one account logged in for your Nextcloud and open a link like https://your.cloud.tld/index.php/f/idnumber")

**Comment by AndyScherzinger** [comment]
Whoops, okay, then this should be fixed of course 👍

**Comment by Shagequi** [comment]
@tobiasKaminsky Good afternoon, do you think we need to merge this PR ? Or we may close it ?

**Comment by ezaquarii** [comment]
Whatever we do here, UI code will be affected by this refactoring:
https://github.com/nextcloud/android/pull/6121
